src/browsers/index.ts

```diff
@@ -436,16 +437,17 @@ export class BrowserManager {
       arg.includes('--proxy-server='),
     );
 
-    if (
-      launchOptions.args &&
-      proxyServerArg &&
-      req.parsed.pathname.startsWith('/playwright')
-    ) {
-      (launchOptions as BrowserServerOptions).proxy = {
-        server: proxyServerArg.split('=')[1],
-      };
-      const argIndex = launchOptions.args.indexOf(proxyServerArg);
-      launchOptions.args.splice(argIndex, 1);
```

Comment:
Not sure why this was formatted differently? I can't see a functional difference.